addons/Assessments_Navigation_Bar/src/presenter.js

```diff
@@ -639,10 +639,37 @@ function AddonAssessments_Navigation_Bar_create(){
         this.$sections;
         this.buttons = [];
         this.shiftCount = 0;
+        this.leftOffset = 0;
 
         this.initView();
     };
 
+    presenter.NavigationManager.prototype.calculateLeftOffset = function (previousLeftSideIndex) {
+        const currentIndex = presenter.playerController.getCurrentPageIndex();
+        const rightSideIndex = previousLeftSideIndex + presenter.configuration.numberOfButtons - 4;
+
+        if (rightSideIndex >= currentIndex) {
+            this.setLeftOffset(1);
```

Comment:
Proszę unikać magic numbers. Obecnie leftOffset może przyjmować wartości 0 lub 1. 
Dlatego proponuję stałego dictionary z stałymi np.
```
const OFFSET {
  NO_OFFSET: 0,
  STANDARD_OFFSET: 1
}
```
Ułatwi to czytanie kodu, bo później można przyrównywać leftOffset z jedną z stałych.